taui/src/components/edit-profile.js

```diff
@@ -486,6 +487,7 @@ export default class EditProfile extends PureComponent<Props> {
                 type='text'
                 onChange={(e) => changeField('headOfHousehold', e.currentTarget.value)}
                 defaultValue={headOfHousehold || ''}
+                autoComplete='off'
```

Comment:
Good call changing all of these as well -- counselors won't want to auto-fill these and individuals shouldn't need to.